addon/transitions.js

```diff
@@ -0,0 +1,31 @@
+export default function () {
+  this.transition(
+    this.inHelper('liquid-modal'),
+    this.toModal(/(?:edit-|create-)/),
```

Comment:
So these would be `edit-user`, `create-user`, I think I'd prefer this to follow more pod based thinking where the category leads, followed by specifics - e.g. `user-*`, which would switch the edit and create to a suffix.  Sound ok?